roles/alexandria-v2/tasks/adrl-dm.yml

```diff
@@ -1,18 +1,18 @@
 ---
-- file: path=/opt/ingest/metadata state=directory
+- file: path={{ project_base }}/ingest/metadata state=directory
 
-- name: clone adrl-dm repository to /opt/ingest/
+- name: clone adrl-dm repository to {{ project_base }}/ingest/
   git: repo=https://{{ ad_git_user }}:{{ ad_git_pass }}@stash.library.ucsb.edu/scm/cms/adrl-dm.git \
-       dest=/opt/ingest/metadata/adrl-dm
+       dest={{ project_base }}/ingest/metadata/adrl-dm
   no_log: True
 
 - name: remove password from remote url
   command: git config remote.origin.url https://stash.library.ucsb.edu/scm/cms/adrl-dm.git \
-           chdir=/opt/ingest/metadata/adrl-dm
+           chdir={{ project_base }}/ingest/metadata/adrl-dm
 
 - name: ensure {{ deploy_user }} owns ingest directories
   file: path={{ item }} owner={{ deploy_user }} group={{ deploy_group }} \
         state=directory recurse=yes
   with_items:
-    - /opt/download_root
-    - /opt/ingest/metadata
+    - "{{ project_base }}/download_root"
```

Comment:
If you use the etd_download_root variable here, then any override to that variable will change both the directory and the config pointing to it.